RELEASE_NOTES.md

```diff
@@ -11,6 +11,10 @@
 ## New
 - Added support for 6 more locales: Catalan (ca-ES), Icelandic (is-IS), Bulgarian (bg-BG),
   Estonian (et-EE), Latvian (lv-LV) and Lithuanian (lt-lT).
+- French meal vouchers are now available with the following payment method types: 
```

Comment:
Can we link to the documentation just like with twint?

Reply:
Done.